src/freva/_user_data.py

```diff
@@ -239,7 +239,7 @@ def index(
         self,
         *crawl_dirs: os.PathLike,
         dtype: str = "fs",
-        continue_on_errors: bool = False,
+        abort_on_errors: bool = True,
```

Comment:
Why did you change that? I think it was better before?

Reply:
I changed after the talk we had in regards of being consistent between python and cli syntax, as this parameter is named `abort_on_errors` under python. But I can backpedal it without any problem.

Reply:
Ok, I thought that but, the `abort_on_erros` flag is passed to the  "internal" API. It's not really supposed to be used by the users, meaning anything that lives outside the `freva` namespace a.k.a `evaluation_system` can be seen as internal. Here we don't have to worry about consistency because it's not supposed to be exposed to the users. So it's totally fine to do what you did in the first place.

On the other hand with the unique output thingy, it's slightly different cos the cli would differ from the user `run_pulgin` method. And that's an inconsistency.  Does that make sense?

Reply:
thanks for the explanation, yes it does make sense. I will make the changes asap.